rfcs/0000-template.md

```diff
@@ -49,7 +49,6 @@ A few examples of what this can include are:
 - Does this feature exist in other programming languages and what experience have their community had?
 - Papers: Are there published papers, books, great blog posts, etc that discuss this? Be _specific_!
 
-
 # Unresolved questions
```

Comment:
Probably better to not have a this arbitrary whitespace change in the template (people might wonder what changed in the template, take the time to check, and discover the answer is "nothing").